src/pages/components/content-group-banner.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Content group – banner
+description: Content group &mdash banner is designed as a subtle callout, intended for tangential information or announcements supplementary to the page's main focus.
```

Comment:
should there be a comma after "announcements" ? 

Content group &mdash banner is designed as a subtle callout, intended for tangential information or announcements, supplementary to the page's main focus.

or

Content group &mdash banner is designed as a subtle callout, intended for tangential information or announcements that are supplementary to the page's main focus.